package.json

```diff
@@ -210,6 +210,7 @@
     "@types/jsonwebtoken": "^8.5.6",
     "@types/mapbox__vector-tile": "1.3.0",
     "@types/moment-duration-format": "^2.2.3",
+    "@types/react-is": "^17.0.3",
```

Comment:
It seems to me that the version of the library and types should be consistent. We use: "react-is": "^16.8.0".

Reply:
Very good point

Reply:
I'd like to see `yarn install` match typings installations to the current package version automatically.